CmdStarter.Tests/Options.cs

```diff
@@ -126,5 +128,42 @@ public void MethodSetup()
                 Assert.That(option, Is.Null);
             }
         }
+
+        [TestCase<OptByAttribute>]
+        [TestCase<OptByIncludeAttribute>]
+        [TestCase<OptByExcludeAttribute>]
+        public void EnsureOptionAttributes<OptClass>() where OptClass : class, IStarterCommand, IOptByAttribute
+        {
+            starter.Namespaces = starter.Namespaces.Add(typeof(OptClass).Namespace!);
+            starter.InstantiateCommands();
+
+            var optionCommand = starter.FindCommand<OptClass>();
+            Assert.That(optionCommand, Is.Not.Null);
+
+            foreach (var includedOption in OptClass.IncludedOptions)
```

Comment:
Couldn't we use `TestsCommon.AssertIEnumerablesHaveSameElements` method instead?

Reply:
Done